templates/music/music.html

```diff
@@ -17,7 +17,7 @@
             {% for album in albums %}
                 <div class="column">
                     <div class="artist-list-image img-list">
-                        {% include "object_preview.html" with object=album img=album.artwork.img %}
+                        {% include "object_preview_music.html" with object=album img=album.artwork.img %}
```

Comment:
Instead of copying the `object_preview.html` code, maybe we can generalize it more so that it can be reused for different object previews. (That's why we made it a separate include in the first place).

What did you change in `object_preview_music.html` that you want to maintain?